website/docs/advanced/strategy-constraints.md

```diff
@@ -5,52 +5,167 @@ title: Strategy Constraints
+
+| Name      | `true` if `<context-field>` is ...                                              |
+|-----------|--------------------------------------------------------------------------------|
+| `NUM_EQ`  | **equal** to the provided value; the mathematical `=` operator                 |
+| `NUM_GT`  | **strictly greater than** the provided value; the mathematical `>` operator    |
+| `NUM_GTE` | **greater than or equal to** the provided value; the mathematical `⩾` operator |
+| `NUM_LT`  | **strictly less than** the provided value; the mathematical `<` operator       |
+| `NUM_LTE` | **less than or equal to** the provided value; the mathematical `⩽` operator    |
+
+You can read more about [numeric equality](https://en.wikipedia.org/wiki/Equality_(mathematics) "Mathematical equality at Wikipedia") or [numeric inequality operators at Wikipedia](https://en.wikipedia.org/wiki/Inequality_(mathematics)).
+
+### Date and time operators
+
+:::info `currentTime` and date and time operators
+The date and time operators are **only available on the `currentTime`** context field. Furthermore, the `currentTime` context field **can not be used** with any of the other operators.
+:::
+
+With the date and time operators, you can enable a feature before and/or after a specified time.
+The operators compare the [Unleash context's](../user_guide/unleash_context) `currentTime` property against the provided value.
+
+You can create a **time span** by combining the two constraint operators using two different constraints on the same strategy.
+In that case the strategy will be evaluated from `DATE_AFTER` and until `DATE_BEFORE`.
+
+Date and time operators only support single values.
+
+| Name          | `true` if `currentTime` is ... |
+|---------------|--------------------------------|
+| `DATE_AFTER`  | **after** the provided date       |
+| `DATE_BEFORE` | **before** the provided date      |
+
+### String operators
+
+:::note Legacy note
+For backwards compatibility reasons, `IN` and `NOT_IN` are always case-sensitive.
+:::
+
+String operators differ from the other categories in two different ways:
+- all operators accept multiple values
+- most operators also consider [letter case](https://en.wikipedia.org/wiki/Letter_case "letter case on Wikipedia") and can be set to be case-sensitive or case-insensitive
+
+| Name              | `true` if `<context-field>` ...                | Available since |
+|-------------------|------------------------------------------------|-----------------|
+| `IN`              | is **equal** to any of the provided values     | v3.3            |
+| `NOT_IN`          | is **not equal** to any of the provided values | v3.3            |
+| `STR_CONTAINS`    | **contains** any of the provided strings       | v4.9            |
+| `STR_ENDS_WITH`   | **ends** with any of the provided strings      | v4.9            |
+| `STR_STARTS_WITH` | **starts** with any of the provided strings    | v4.9            |
```

Comment:
I think we should point out which operators support case_insensitive here. For Python, Ruby, Rust, Node and Go that's STR_CONTAINS, STR_ENDS_WITH and STR_STARTS_WITH (Java seems to have added support for IN and NOT_IN too but I think it's an outlier and this isn't encoded in the spec).

Reply:
Got it! I've mentioned it above, but I agree that adding it to the table may be better.